README.md

```diff
@@ -233,6 +233,10 @@ You can specify a different branch, commit or tag using the `baseBranch` variabl
 crank --config benchmarks.yml --scenario Service --profile local --variable base=true --variable baseBranch=v7.6.4
 ```
 
+#### Comparing benchmarks
+
+You can use the [ResultsComparer](./tools/perf/ResultsComparer) tool to compare benchmark results.

```

Comment:
Kindly add what the parameters for this are, e.g to compare results use 

```sh
./ResultsComparer [ and any required params]
```

Reply:
The linked repo contains a readme with instructions and params to use. But let me also add some examples here.

Reply:
Done.